server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -3159,7 +3159,8 @@ private void returnMasterConnection(MasterClientService.Client client) {
 
   private HostAndPort startTabletClientService() throws UnknownHostException {
     // start listening for client connection last
-    Iface tch = RpcWrapper.service(new ThriftClientHandler());
+    ThriftClientHandler handler = new ThriftClientHandler();
+    Iface tch = RpcWrapper.service(handler, new Processor<Iface>(handler).getProcessMapView());
```

Comment:
For all of these calls, I think it's better to pass the Processor, and call the `getProcessMapView()` in the method. It makes it a lot more readable, especially with all the generics.